content/waku/2024-gantt.md

```diff
@@ -24,92 +24,128 @@ gantt
-        Store v3 (sync) research + RFC: crit, storev3-r, 2024-03-25, 14w
+        Store v3 (sync): crit, storev3sync-r, 2024-08-01, 2024-09-20
         Store v3 - follow-up: after storev3-r, 8w
         Peer mgmt - follow-up: after storev3-r, 8w
-    section nim eng 1
-        PostgreSQL Maintenance: crit, pgsql, 2024-01-01, 2024-07-31
-        Reliability Protocol for Relay (nwaku + RFC): crit, rel-relay-n, 2024-06-19, 12w
-    section nim eng 2
-        DoS Protection for Req-Res Protocols: crit, dosreqresn, 2024-02-01, 18w
-        (tooling) filter and light push protocols: lite-proto-tester, 2024-05-13, 2024-07-03
-        Store v3-beta + v3 (dogfooding placeholder):  storev3-df, after storev3-br storev3-r, 4w
-        %% More hardening expected to deprecate or separate store v2 from v3 driver
+        Usage of rendezvous: rendezvous-r, after storev3sync-r, 8w
+    section Res.alrevuelta
+    section Res.s-tikhomirov
+        Implement RLN smart contract for paid, multilevel memberships: rln-sc-d, 2024-09-01, 6w
+        %% TODO Estimate
+        Pay for RLN provision first PoC: rlnaas-r, 2024-09-01, 8w
+    section Nim.Ivansete
+        Store v3 (sync): crit, storev3sync-n, 2024-08-15, 2024-09-20
+        PostgreSQL Maintenance: crit, pgsql-m, 2024-08-01, 2024-09-18
+        %% TODO review date/estimate
+        PostgreSQL Optimisation - phase 1: pgsql-opt-1, after pgsql-m, 8w
+        PostgreSQL Optimisation - phase 2: pgsql-opt-2, after pgsql-opt-1, 8w
+        %% TODO: Estimate
+        Nwaku in Status Desktop: nwaku-status-desktop-2, after pgsql-opt-1, 6w
+    section Nim.NagyZoltanPeter
+        DoS Protection for Req-Res Protocols: crit, dosreqres-n, 2024-08-01, 2024-09-15
+        %% TODO review
         TBC Store v3-beta + v3 (nwaku hardening): crit, storev3-n, after storev3-df dosreqresn, 3w
-        (telemetry) Fleet logging: telem-fleet-logging, after dosreqresn, 4w
-    section nim eng 3
-        Sharding peer mgmt and discovery hardening (nwaku): crit, sh-peer-mgmt-n, 2024-05-13, 12w
-    section js eng 1
-        Reliability for Req-Res Protocols (light client + RFC): crit, rel-reqres-j, 2024-05-01, 12w
-    section js eng 2
-        %% Buidling idle apps and integrating in telemetry service to learn
-        Reliability for Req-Res Protocols (light client + RFC): crit, rel-reqres-j, 2024-05-01, 12w
-        Sharding peer mgmt and discovery hardening (light client + RFC): crit, sh-peer-mgmt-j, 2024-06-01, 12w
-    section js eng 3 (dev rel)
-        %% TBC timing and estimate
-        TBC User apps for large scale dogfooding - GUI and Gamification: user-app-gui, 2024-06-01, 4w
+        Nwaku in Status Desktop: nwaku-status-desktop-n, after storev3-n, 12w
+    section Nim.gabrielmer
+        Sharding peer mgmt and discovery hardening: crit, sh-peer-mgmt-n, 2024-08-01, 2024-09-12
+    section Nim.darshankabariya
+        %% TODO: review estimate
+        Nwaku on Windows: nwaku-windows-n, 2024-08-15, 6w
+    section Js.weboko
```

Comment:
Are `rln v2` and API stabilization efforts for later?

Reply:
Good catch! Fixed. Can you please help provide estimate for all js-waku items.

Reply:
Thanks for adding!

Won't we have `API stabilization` as a separate work stream? Though we discussed it - it is vaguely defined as we don't have any backlog for it. We can add it at the end of H2 as well so not to overcommit. 

For estimates for `dogfooding` app I think `8w` is good as it is aligned with beginning of Devcon. 

For `reliability` all good as per https://github.com/waku-org/pm/issues/186#issuecomment-2322916557

For `rln v2` - I think `8w` is good estimate but I am not sure as for the scope (in my mind it seems quite straightforward to do) so if possible I'd add `2w` on top, if no we can go with current and extend if it is needed.

Reply:
Let's continue the planning discussion in discord